EPFL.md

```diff
@@ -0,0 +1,38 @@
+# EPFL
+
+An instance of this app is deployed at EPFL under the name Palabrate.
+
+Webapp url: https://modemos.epfl.ch/palabrate
+
+## Deploy Firebase
+
+You must have logged in to the Firebase CLI with the account that has access to this project using the command `firebase login`.
+
+```bash
+firebase deploy  # Deploys everything (Cloud Functions, Firestore rules)
+firebase deploy --only functions  # Deploys only Cloud Functions
+```
+
+You can then access various consoles at the following URLs:
+
+| Console                 | URL                                                                                                       |
+| ----------------------- | --------------------------------------------------------------------------------------------------------- |
+| Project Console         | https://console.firebase.google.com/u/0/project/friendlychat-d6dc5/overview                               |
```

Comment:
Lets try and factor our project IDs from documentation also. (i.e. there should be no mention of friendlychat-d6dc5 in the repo)

Reply:
done in #8 - 6c9d812932eade4f8ccee8fca2dd83da0f103edb